doc/release-notes/release-7.sgml.in

```diff
@@ -123,6 +124,13 @@ in the position of what used to be a %ui record field.
 <p>If necessary, an external ACL helper can be written to perform Ident transactions
 and deliver the user identity to Squid through the **user=** annotation.
 
+<sect1>Remove custom log location for ext_time_quota_acl helper
```

Comment:
For consistency with other `<sect1>` titles above:

```suggestion
<sect1>Removed custom log location for ext_time_quota_acl helper
```

I would also clarify with:

```suggestion
<sect1>Removed custom log location support from ext_time_quota_acl helper
```

... and even generalize since we mention formatting changes:

```suggestion
<sect1>Adjusted configuration and format of ext_time_quota_acl helper debugging
```

Reply:
adopting the third option